compat/visual-composer/siteorigin_widget_vc_template.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$widget_settings = $this->get_widget_settings( $atts );
+$instance = $this->update_widget( $widget_settings['widget_class'], $widget_settings['widget_data'] );
+$this->render_widget( $widget_settings['widget_class'], $widget_settings['widget_data'] );
+?>
```

Comment:
Do we need this closing PHP tag in here? Generally, it's best to remove this to prevent any accidental white space after the file.